src/jvmTest/java/org/snakeyaml/engine/usecases/fuzzy/FuzzYAMLRead56902Test.java

```diff
@@ -13,20 +13,6 @@
  */
 package org.snakeyaml.engine.usecases.fuzzy;
 
-/**
- * Copyright (c) 2008, SnakeYAML
- * <p>
- * Licensed under the Apache License, Version 2.0 (the "License"); you may not use this file except
- * in compliance with the License. You may obtain a copy of the License at
- * <p>
- * http://www.apache.org/licenses/LICENSE-2.0
- * <p>
- * Unless required by applicable law or agreed to in writing, software distributed under the License
- * is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express
- * or implied. See the License for the specific language governing permissions and limitations under
- * the License.
- */
```

Comment:
The license info is duplicated in this file, this change removes the copy. It's unrelated to this PR, but it's a small fix.